core/templates/expressions/expression-syntax-tree.service.spec.ts

```diff
@@ -15,40 +15,30 @@
 /**
  * @fileoverview Unit tests for expression-syntax-tree.service.ts
  */
-
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// the code corresponding to the spec is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-require('App.ts');
-require('expressions/expression-syntax-tree.service.ts');
+import { ExpressionParserService } from
+  'expressions/expression-parser.service.ts';
+import { ExpressionSyntaxTreeService } from
+  'expressions/expression-syntax-tree.service.ts';
 
 describe('Expression syntax tree service', () => {
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-
   describe('expression syntax tree service', () => {
-    let ExpressionSyntaxTreeService = null;
+    let expressionSyntaxTreeService: ExpressionSyntaxTreeService;
 
-    beforeEach(angular.mock.inject(($injector) => {
-      ExpressionSyntaxTreeService =
-          $injector.get('ExpressionSyntaxTreeService');
-    }));
+    beforeEach(() => {
+      expressionSyntaxTreeService = new ExpressionSyntaxTreeService(
```

Comment:
This is fine but can we please use TestBed to create an instance of this service please? This will make our code consistent and that is a more cleaner way of creating an instance. 

Think what is ExpressionParserService() also has some dependency, we would have so many nested calls.